src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -57,7 +58,7 @@ private partial class WorkCoordinator
                 var entireProjectWorkerBackOffTimeSpanInMS = _optionService.GetOption(InternalSolutionCrawlerOptions.EntireProjectWorkerBackOffTimeSpanInMS);
 
                 _documentAndProjectWorkerProcessor = new IncrementalAnalyzerProcessor(
-                    listener, analyzerProviders, _registration,
+                    listener, analyzerProviders, initializeLazily, _registration,
```

Comment:
this flag seems pretty strange to me.  not clear to me why a caller would want to control this.

Reply:
see comments in the code.